manifests/init.pp

```diff
@@ -18,6 +18,7 @@
 #
 class phpfpm (
   $ensure                      = 'present',
+  $pools                       = $phpfpm::params::pools,
```

Comment:
`$phpfpm::params::pools` is not defined anywhere. Maybe this should be changed to `$pools = undef`? 

Then the `create_resources()` command can be surrounded by an `if` statement, for example: `if $pools { create_resources(...) } `